reana_commons/validation/schemas/reana_analysis_schema.json

```diff
@@ -200,5 +204,127 @@
         }
       }
     }
+  },
+  "if": {
+    "properties": {
+      "workflow": {
+        "properties": {
+          "type": {
+            "const": "serial"
+          }
+        }
+      }
+    }
+  },
+  "then": {
+    "properties": {
+      "workflow": {
+        "properties": {
+          "specification": {
+            "type": "object",
+            "title": "Serial workflow specification.",
+            "description": "Serial workflow specification.",
+            "additionalProperties": false,
+            "properties": {
+              "steps": {
+                "type": "array",
+                "title": "Serial workflow steps.",
+                "description": "List of steps which represent the workflow.",
+                "items": {
+                  "type": "object",
+                  "title": "Serial workflow step.",
+                  "description": "Serial workflow step.",
+                  "additionalProperties": false,
+                  "properties": {
+                    "commands": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      },
+                      "minItems": 1,
+                      "title": "Step commands",
+                      "description": "List of commands to be run in the step."
+                    },
+                    "compute_backend": {
+                      "type": "string",
+                        "enum": [
+                          "kubernetes",
+                          "htcondor",
+                          "htcondorcern",
+                          "slurm",
+                          "slurmcern"
```

Comment:
I do not think `slurm` and `htcondor` are allowed here.  Did you find any places where they are used?

Reply:
I put them there to be less restrictive towards other REANA deployments that might want to integrate HTCondor or Slurm, but I don't know how the integration with those computational backends would work if external to CERN. Do you think it is better to remove them (and also avoid that CERN users mistakenly write `"htcondor"` instead of `"htcondorcern"`)?

Reply:
> and also avoid that CERN users mistakenly write "htcondor" instead of "htcondorcern"

Yes, exactly, I think we should avoid this so let's remove them as currently we only support `kubernetes`, `htcondorcern` and `slurmcern`. When we will add support for other Slurm/HTCondor instances we will update the schema accordingly!